Server/Components/Dialogs/dialog.cpp

```diff
@@ -167,7 +175,7 @@ class DialogsComponent final : public IDialogsComponent, public PlayerConnectEve
 				&PlayerDialogEventHandler::onDialogResponse,
 				peer,
 				sendDialogResponse.ID,
-				static_cast<DialogResponse>(sendDialogResponse.Response),
+				static_cast<DialogResponse>(sendDialogResponse.Response % 2),
```

Comment:
Wouldn't it be better to skip dialog handling for dialogs with Response not 0 or 1?